SixTrack/sixtrack.s

```diff
@@ -3890,50 +3892,55 @@
           pi=4d0*atan(1d0)
 +ei
         crabfreq=ek(ix)*c1e3
-
+        crabamp=ed(ix)
```

Comment:
This variable `crabamp` no longer serves any purpose, why not delete it?

Reply:
Sure I can delete it. It was left for almost like a comment for what ed(ix) actually is.